source/MapPanel.cpp

```diff
@@ -346,6 +346,7 @@ void MapPanel::DrawMiniMap(const PlayerInfo &player, float alpha, const System *
 			RingShader::Draw(to, OUTER, INNER, color);
 		}
 
+		DistanceMap localDistance(player);
```

Comment:
Will this account for the jump capabilities of hte player's flagship and what systems/links they know about?